pmg/templates/committee_detail.html

```diff
@@ -117,7 +118,8 @@ <h3 class="no-results">No results.</h3>
           {% endif %}
         </div>
         <div class="tab-pane fade in cte-dtl-meetings-tab-rep" id="tr">
-          {% if committee.tabled_committee_reports %}
+	   {% include '_active_committee_detail.html' %}
```

Comment:
the notice above is specific to calls and the ministry while the rest of the tabs seem to be non-question content about committees instead - let's do more appropriate wording for them